README.md

```diff
@@ -14,7 +14,7 @@ pip install ftdc-tools
 
 ### Decoding FTDC file from URL - Streaming approach
 ```
-from ftdc_tools.ftdc_decoder import FTDC
+from ftdc_tools.decoder import FTDC
```

Comment:
Having moved ftdc_tools under src should the import be `from src.ftdc_tools.decoder import FTDC`

Reply:
Ignore this comment. The import statement is correct. Approving this PR